content/docs/hooks-state.md

```diff
@@ -258,23 +266,27 @@ function ExampleWithManyStates() {
   const [todos, setTodos] = useState([{ text: 'Learn Hooks' }]);
 ```
 
-In the above component, we have `age`, `fruit`, and `todos` as local variables, and we can update them individually:
+Yukarıdaki bileşende, `age`, `fruit` ve `todos`'a yerel değişken olarak sahibiz ve bunları teker teker güncelleyebiliriz.
 
 ```js
   function handleOrangeClick() {
     // Similar to this.setState({ fruit: 'orange' })
+    // bu this.setState({ fruit: 'orange' })'a benzer.
     setFruit('orange');
   }
 ```
 
 You **don't have to** use many state variables. State variables can hold objects and arrays just fine, so you can still group related data together. However, unlike `this.setState` in a class, updating a state variable always *replaces* it instead of merging it.
+bir çok state değişken **kullanmak zorunda değilsiniz**. State değişkenleri, objeler ve dizileri çok iyi tutabilr bu yüzden ilgili verileri birlikte tutabilirsiniz fakat sınıftaki `this.setState`'ın aksine state değişkenini güncellemek birleştirmek(merge) yerine *değiştirir*.
+
+Bağımsız değişkenleri bölme konusunda daha çok fazla öneriye bu bölümden ulaşabilirsiniz [in the FAQ](/docs/hooks-faq.html#should-i-use-one-or-many-state-variables).
```

Comment:
Bağımsız state değişkenlerini ayirma konusunda daha fazla öneriye `[bu bölümden](/docs/hooks-faq.html#should-i-use-one-or-many-state-variables)` ulaşabilirsiniz.